### DISCUSSION TITLE
Intended functionality

### DISCUSSION THREAD

**Discussion by Taxit0m** [hardalgorithm/ParanoidsPal-PrivacyLock#4] [Q&A] [answered]
Can you elaborate on the intended functionality of this application and the differences between androids included versus the version you are offering?   
  
  Does your application perform anything not already included with android?  If so, what are they and why are they different or offer superior security?

**Comment by hardalgorithm** [comment, marked as answer]
https://github.com/hardalgorithm/ParanoidsPal-PrivacyLock/discussions/3#discussioncomment-11077610
